### DISCUSSION TITLE
Search not searching in filename?

### DISCUSSION THREAD

**Discussion by AcidReign81** [jonaswinkler/paperless-ng#582] [Support]
Hi everybody,

I am going forward testing this awesome software and I have a weird behavior.
The upper searchbar does not search in filenames. I have a filename called "invoice_car" for example. If i search for invoice it does not show up because the word invoice is not inside of the file - only in the filename.
When I filter in the documents page the filename is found without a problem. Only the global search does not find it.
Is this a setup problem?

Thank you

**Comment by jonaswinkler** [comment]
When you say filename, are you referring to the filename as it is displayed in the Metadata tab next to "media filename", or are you referring to the title of the document as displayed on the document list?

**Reply by AcidReign81** [reply]
actually the word invoice is in the title and in die mediafilename in the metadata both
